test/envtest/qatdeviceplugin_controller_test.go

```diff
@@ -131,11 +134,24 @@ var _ = Describe("QatDevicePlugin Controller", func() {
 				"-allocation-policy",
 				updatedPreferredAllocationPolicy,
 			}
+			mode := int32(420)
```

Comment:
It's time to fix the last missing item from #819 and get rid of this mode.

Reply:
made mode have 292 (444 in octal)